tests/copyright/policy

```diff
@@ -178,6 +178,7 @@ modules/affile/tests/test_file_writer\.c$
 modules/affile/stdout\.[ch]
 modules/afsocket/socket-options-unix\.[ch]$
 modules/basicfuncs/vp-funcs\.[ch]$
+modules/basicfuncs/tf-tag\.c$
```

Comment:
I believe this should go to the LGPL section.

Reply:
It was actually in the right section, but the license header was incorrect. I meant to license it under the GPL, as it's usual for modules.

Fixed this up.